gen/_template/client.tmpl

```diff
@@ -304,7 +304,6 @@ func (c *{{ if $op.WebhookInfo }}Webhook{{ end }}Client) send{{ $op.Name }}(ctx
 	if err != nil {
 		return res, errors.Wrap(err, "do request")
 	}
-	defer resp.Body.Close()
```

Comment:
I think client should close response body if any error occurred.

So this line should be 
```go
defer func() {
       if err != nil {
            resp.Body.Close()
       }
}()
```
instead.